src/Synergism.ts

```diff
@@ -782,7 +782,7 @@ const loadSynergy = async () => {
         ? JSON.parse(atob(saveString)) as PlayerSave & Record<string, unknown>
         : null;
 
-    if (testing) {
+    if (!testing) {
```

Comment:
revert

Reply:
Thank you. It is already included in my next correction